packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/get-form-tool.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+import { GetFormQuery, GetFormQueryVariables } from '../../../../monday-graphql/generated/graphql';
+import { getForm } from './workforms.graphql';
+import { ToolInputType, ToolOutputType, ToolType } from '../../../tool';
+import { BaseMondayApiTool, createMondayApiAnnotations } from '../base-monday-api-tool';
+import { GraphQLDescriptions } from './workforms.consts';
+
+export const getFormToolSchema = {
+  formToken: z.string().describe(GraphQLDescriptions.commonArgs.formToken),
+};
+
+export class GetFormTool extends BaseMondayApiTool<typeof getFormToolSchema, never> {
+  name = 'get_form';
+  type = ToolType.READ;
+  annotations = createMondayApiAnnotations({
+    title: 'Get Form',
+    readOnlyHint: true,
+    destructiveHint: false,
+  });
+
+  getDescription(): string {
+    return 'Get a monday.com form by its form token. Form tokens can be extracted from the form’s url. Given a form url, such as https://forms.monday.com/forms/abc123def456ghi789?r=use1, the token is the alphanumeric string that appears right after /forms/ and before the ?. In the example, the token is abc123def456ghi789.';
```

Comment:
In general, the MCP should work like there is no connection to the UI, so if user is expected to go and get this form tokem, how will the flow look like?

Or whats the use case we expect to be used here?